localization/english/religion/wc_religion_l_english.yml

```
@@ -5,4 +5,502 @@
+ 
+ shamanism:0 "Gwyari"
+ shamanism_adj:0 "Gwyari"
+ shamanism_adherent:0 "Gwyari"
+ shamanism_adherent_plural:0 "Gwyari"
+ shamanism_desc:0 "A shamanic faith founded by the Wildhammer dwarves. Like many $shamanism_group_adherent_plural$, they believe that their ancestors watch over them. Burial rituals and ancestor reverence are the cornerstone of religion. Family ties between living family members are equally important."
+ 
+ earth_mother_worship:0 "Mashani"
+ earth_mother_worship_adj:0 "Mashani"
+ earth_mother_worship_adherent:0 "Mashani"
+ earth_mother_worship_adherent_plural:0 "Mashani"
+ earth_mother_worship_desc:0 "A religion worshipping the Earth Mother, a deity representing not only the earth but nature itself. Thus the $earth_mother_worship$ connects druidic and shamanic practices. Apart from the Earth Mother, interactions with the spirit world and ancestors are the main concepts of faith."
+ 
+ tidemother:0 "Tirani"
+ tidemother_adj:0 "Tirani"
+ tidemother_adherent:0 "Tirani"
+ tidemother_adherent_plural:0 "Tirani"
+ tidemother_desc:0 "The Tidemother is a being worshipped by many sailors and those whose lives depend on mutable sea winds. Most people look to the seas for guidance the same way others look to the Light, but the tidesages, the priest of the Tidemother, have a much deeper connection with the water. Long have the tidesages listened to the sea to guide their ships and their people, hearing gentle whispers imperceptible to most."
+ 
+ cult_of_the_candle:0 "Candlism"
+ cult_of_the_candle_adj:0 "Candlic"
+ cult_of_the_candle_adherent:0 "Candlist"
+ cult_of_the_candle_adherent_plural:0 "Candlists"
+ cult_of_the_candle_desc:0 "Who would have thought that even such a simple thing as a candle could become a symbol of faith? Kobolds that dwell in underground tunnels and spend their lives mining, primarily for gold, are sure that the candle is wise. And this is not surprising, because in the condition of darkness it is the candle that helps the $cult_of_the_candle_adherent_plural$ to survive."
+ 
+ primitive_shamanism:0 "Fetishism"
+ primitive_shamanism_adj:0 "Fetishistic"
+ primitive_shamanism_adherent:0 "Fetishist"
+ primitive_shamanism_adherent_plural:0 "Fetishists"
+ primitive_shamanism_desc:0 "Faith in lesser known elementals and spirits of the world. $primitive_shamanism_adj$ beliefs encompass faiths closely guarded by their peoples. Rituals, wards, and other simplistic practices are employed in worship to their patrons. Worshiper’s desire some sort of balance with the natural world which binds these primitive beliefs together categorically. This category of faith only differs from other shamanistic faiths by the amount of known information. As further information is uncovered by explorers and compiled, the amount of primitive shamanistic beliefs in Azeroth decreases."
```

Changed the first sentence a little.
```suggestion
 primitive_shamanism_desc:0 "Fetishism is faith in lesser known elementals and spirits of the world. $primitive_shamanism_adj$ beliefs encompass faiths closely guarded by their peoples. Rituals, wards, and other simplistic practices are employed in worship to their patrons. Worshipers desire some sort of balance with the natural world which binds these primitive beliefs together categorically. This category of faith only differs from other shamanistic faiths by the amount of known information. As further information is uncovered by explorers and compiled, the amount of primitive shamanistic beliefs in Azeroth decreases."
```